doc/holochain_101/src/zome/api_functions.md

```diff
@@ -137,39 +137,47 @@ Entry lookup is done in the following order:
 Caller can request additional metadata on the entry such as type or sources
 (hashes of the agents that committed the entry).
 
-[View it in the Rust HDK](https://developer.holochain.org/api/latest/hdk/api/fn.get_entry.html)
+- [View get_entry in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry.html)
+- [View get_entry_initial in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry_initial.html)
+- [View get_entry_initial in the Rust HDK](https://developer.holochain.org/api/0.0.3/hdk/api/fn.get_entry_initial.html)
```

Comment:
looks like this line is in there twice.

Reply:
Ah, thanks. That should say 'get_entry_history'